codalab_service.py

```diff
@@ -177,7 +179,7 @@ def has_callable_default(self):
     CodalabArg(
         name='bundle_mount',
         help='Path to bundle data (just for mounting into Docker)',
-        default='/tmp',
+        default=var_path(''),  # Put any non-empty path here
```

Comment:
Why did this change? Did this have to do with making worker managers work?

Reply:
It's kind of unrelated...sharing a /tmp directory just pollutes the host's tmp, so it was a bad choice.